### ISSUE TITLE
Game allows keyboard to have a repeat rate. Contributes to #91, #92 and #94

### ISSUE THREAD

**Issue by SapphireTactician** [eliatlarge/FEMultiPlayer-V2#93]
Issue type: `problem`

#### Description:

Keys will "rapid fire" when held in. This contributes to #91 and #92 considerably. But these are all just symptoms that uncover #94

**Comment by rayrobdod** [comment]
Can you rephrase this in the form of an accessibility issue? Because this seems simple enough for the sake of accessibility, but not for the sake of not actually fixing anything. And even then repeat rate is more an OS thing - I know Windows has a pair of settings to change repeat rate and if Q/As from 2008 can be trusted, so does Linux.

**Comment by SapphireTactician** [comment]
I'm aware that it's an OS feature, however, the game itself doesn't have to honor it is what I'm saying. The best way I can explain it, is that by defining how the application handles "key events" you can override things such as the maximum repeat rate, having a repeat rate at all, and even detecting when a key is pressed or depressed for the game window. The OS will still do it's thing, but the game will simply filter it as defined by the key events. I'll probably have to ask my brother, who is a programmer and a damn stellar one, how he went about this for the games he programs.

**Comment by SapphireTactician** [comment]
Closing as `Won't Fix` - Once finished, Pull #102 will make fixing this completely unnecessary, plus it helped uncover and confirm nightmares #91 #92 and #94
